spec/support/idv_examples/failed_idv_phone_job.rb

```diff
@@ -1,42 +1,33 @@
-shared_examples 'failed idv job' do |step|
```

Comment:
There was no call to the shared example with any step but `:phone` so I removed all the conditional code.